src/peft/utils/config.py

```diff
@@ -42,13 +133,11 @@ class PeftConfig:
         task_type (Union[[`~peft.utils.config.TaskType`], `str`]): The type of task to perform.
         inference_mode (`bool`, defaults to `False`): Whether to use the Peft model in inference mode.
     """
-
     peft_type: Union[str, PeftType] = field(default=None, metadata={"help": "Peft type"})
     task_type: Union[str, TaskType] = field(default=None, metadata={"help": "Task type"})
     inference_mode: bool = field(default=False, metadata={"help": "Whether to use inference mode"})
 
 
-@dataclass
```

Comment:
This is required else similar error as seen in the previous comment

Reply:
Thanks! In any case I added tests to test that the script you shared above will not fail